druid-derive/src/attr.rs

```diff
@@ -105,52 +109,75 @@ impl Field {
         let mut same_fn = None;
         let mut lens_name_override = None;
 
-        for attr in field
-            .attrs
-            .iter()
-            .filter(|attr| attr.path.is_ident(BASE_ATTR_PATH))
-        {
-            match attr.parse_meta()? {
-                Meta::List(meta) => {
-                    for nested in meta.nested.iter() {
-                        match nested {
-                            NestedMeta::Meta(Meta::Path(path))
-                                if path.is_ident(IGNORE_ATTR_PATH) =>
-                            {
-                                if ignore {
-                                    return Err(Error::new(nested.span(), "Duplicate attribute"));
+        for attr in field.attrs.iter() {
+            if attr.path.is_ident(BASE_DRUID_DEPRECATED_ATTR_PATH) {
+                return Err(Error::new(
+                    attr.path.span(),
+                    "The 'druid' attribute has been replaced with separate \
+                    'lens' and 'data' attributes.",
+                ));
```

Comment:
okay I played around with this a bunch and I think your version was better, where we just panic; returning an error causes compilation to continue, and we get a whole bunch of subsequent error messages. :(

```suggestion
                panic(
                    "The 'druid' attribute has been replaced with separate \
                    'lens' and 'data' attributes.",
                );
```